jbehave-support-core/src/main/java/org/jbehavesupport/core/report/extension/FailScreenshotsReporterExtension.java

```diff
@@ -13,7 +13,7 @@
 public class FailScreenshotsReporterExtension extends AbstractXmlReporterExtension {
 
     private static final String FAIL_SCREENSHOTS_REPORTER_EXTENSION = "errorScreenshots";
-    private static final String SCREENSHOT_TAG = "<screenshot>%s</screenshot>";
+    private static final String SCREENSHOT_TAG = "<screenshot>%s</screenshot>\n";
```

Comment:
Do we need to add empty line?

Reply:
report XML looks better with new line :)

Reply:
I personally still would not do it - if we want it to be more human readable I would rather pretty print the whole report at the end. Better result + less code + less chance of human error (something like https://stackoverflow.com/a/1264872).

But I am also interested in what others think of this 🙂

Reply:
Removed